ext/Caddyfile

```diff
@@ -0,0 +1,32 @@
+# Comment this out if you need to explicitly
+# use self-signed certs.
+# NOTE: Not necessary if using a '.local' domain
+#
+# {
+# 	local_certs
+# }
+
+c.{$DOMAIN}, ce.{$DOMAIN}, c4.{$DOMAIN}, osu.{$DOMAIN}, b.{$DOMAIN}, api.{$DOMAIN} {
+	encode gzip
+	# NOTE: if you wish to use unix sockets,
+	# reverse_proxy * unix//tmp/bancho.sock {
+	reverse_proxy * 127.0.0.1:{$APP_PORT} {
+		header_up X-Real-IP {remote_host}
+	}
+
+	request_body {
+		max_size 5MB
+	}

```

Comment:
maybe nginx should have this too

Reply:
also not 100% sure but screenshots may be larger than this

iirc on akatsuki we went 5->10->100 over time from hitting the limits, but that included other thnigs like avatars/profile backgrounds/etc.